### PR TITLE
Upgrade USWDS version to 1.4.4

### PR DESCRIPTION
**Pull request by donjo**
Upgrades the USWDS package to 1.4.4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -93,7 +93,7 @@
     "politespace": "^0.1.4",
     "run-sequence": "^1.1.5",
     "simplecrawler": "^1.1.3",
-    "uswds": "1.4.3",
+    "uswds": "^1.4.4",
```
Comment (maya) [inline package.json]: Carrot needs to be removed. We usually add this manually, so maybe that step should be removed.

Comment (donjo) [reply package.json]: Ok, will edit the line in the release that says to install via the `npm install --save uswds@{{ version }}` command

**Comment by maya** [comment]
For the future, let's try to use consistent branch names.. i.e. `release-1.4.4`.

**Comment by donjo** [comment]
I propose we change the recommended action in that release step to reference what is being updated since using the word `release` alone implies that this project is receiving a new version when we're just updating a dependency. I don't know if the one I made up on the spot is the best option but I think that `release-1.4.4` would be unclear.

**Comment by maya** [comment]
That works for me, maybe `update-uswds-v1.4.4`, `update-uswds-1.4.4`, or `update-v1.4.4`?

**Pull request merged**